app/modules/sightings/models.py

```diff
@@ -1449,8 +1454,10 @@ def identified(self, job_id, data):
 
                 if not self.any_jobs_active():
                     self.set_stage(SightingStage.un_reviewed)
+                    self.unreviewed_start = datetime.datetime.utcnow()
```

Comment:
This was another bug. The unreviewed start time was being set after the first job completed but the sighting was only reviewed when the last job completed.